src/main/groovy/com/bmuschko/gradle/docker/DockerJavaApplication.groovy

```diff
@@ -18,6 +18,17 @@ package com.bmuschko.gradle.docker
 class DockerJavaApplication {
     String baseImage = 'java'
     String maintainer = System.getProperty('user.name')
+    @Deprecated
     Integer port = 8080
+    Integer[] ports = []
     String tag
+
+    Integer[] getPorts() {
```

Comment:
@cdancy I've thought about this variant, and imho it's not so clean for end users cause they can use port and ports variable simultaneously. And if we want to deprecate port in future release, I don't think that's a good idea (several build script fixes pre and before plugin release).
But if you want, I'll add this.